rapids/README.md

```diff
@@ -0,0 +1,101 @@
+# RAPIDS
+
+This initialization action installs the latest release version of [RAPIDS](https://rapids.ai/) on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster. 
+
+These initialization actions automate the process of setting up a Dask-cuDF cluster by running the following:
+
+On the Dataproc master node:
+- `dask-scheduler`, and `dask-cuda-worker`
+
+On the Dataproc worker nodes:
+- `dask-cuda-worker`
+
+Our initialization actions do the following:
```

Comment:
Consider rewording to: `The initialization action does the following` since there is only 1 init-action used below.

Reply:
done